woocommerce-framework-plugin-loader-sample.php

```diff
@@ -177,7 +177,7 @@ protected function load_framework() {
 	 *
 	 * @return string
 	 */
-	protected function get_framework_version_namespace() {
+	private function get_framework_version_namespace() {
```

Comment:
come to think of it, how about making this method public? Seems like we could provide it in case someone needs to get the namespace dynamically